Common Collections/Hashmaps/More Fruits/task-info.yaml

```diff
@@ -3,14 +3,14 @@ files:
 - name: src/main.rs
```

Comment:
same here: maybe we could place lib.rs before main.rs here in yaml, so that when the student opens this task lib.rs would be shown in the code editor instead of main.rs